### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by junaid-iw**
- Added Nav Bar to template
- Added CSS and Assets
- Fixed issue with login page not showing error message when incorrect password is inputted due to new CSS conflicts.

### CODE REVIEW

**Review by MikeSilverstone — COMMENTED** [review]
.DS_Stores need to be removed from git and added to .gitignore, if they aren't already.

**Review by MikeSilverstone — DISMISSED** [review]

File: globals/templates/index.html [hunk 1]
```diff
@@ -1,18 +1,8 @@
-<!DOCTYPE html>
-<html>
-<head>
-{% include "sections/head.html" %}
-</head>
+{% extends "base.html" %}
```
[the remark this answers is not in the document]
Comment (junaid-iw) [reply globals/templates/index.html]: This page can be replaced or removed, it was the hello world page

File: globals/static/scripts.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    } else {
+        popup.style.display = "none";
+    }
+}
+
+function displaySnailXSidebar() {
+    popup.style.display = "none";
+    if (sidebar.style.display === "none") {
+        sidebar.style.display = "block";
+    } else {
```
[the remark this answers is not in the document]
Comment (junaid-iw) [reply globals/static/scripts.js]: I tried the code without the else statement, but JS doesn't like it :(

**Review by sorsby — APPROVED** [review]
Good stuff.

**Pull request merged**